### ISSUE TITLE
Formula in https://github.com/privefl/paper-infer/blob/main/code/example-with-provided-LD.R

### ISSUE THREAD

**Issue by scienception** [privefl/bigsnpr#531]
Shouldn't `sd_ss <- with(info_snp, 2 / sqrt(n_eff * beta_se^2))` be `sd_ss <- with(info_snp, 2 / sqrt(n_eff * beta_se^2 + beta^2))`?
Isn't +beta^2 missing?

**Comment by scienception** [comment]
Also how would the QC be performed for continuous traits?

```
sd_ldref <- with(info_snp, sqrt(2 * af_UKBB * (1 - af_UKBB)))
sd_ss <- with(info_snp, 1 / sqrt(n_eff * beta_se^2 + beta^2))
sd_ss <- sd_ss / quantile(sd_ss, 0.99) * sqrt(0.5)
is_bad <- sd_ss < (0.5 * sd_ldref) | sd_ss > (sd_ldref + 0.1) | sd_ss < 0.05 | sd_ldref < 0.05
```
I saw in one of your scripts you used sd_ss / sqrt(info), but I'm not sure why.

**Comment by privefl** [comment]
Yes, you're right about the missing `beta^2`, which is important for continuous traits with large effects (e.g. some protein measurements). Thanks

For continuous traits, you need to estimate `sd(y)`, e.g. as explained at https://privefl.github.io/bigsnpr/articles/LDpred2.html#quality-control-of-gwas-summary-statistics.

**Comment by scienception** [comment]
Understood, thanks!
